src/components/Toast/Toast.tsx

```diff
@@ -11,7 +12,7 @@ export type ToastComponentProps = {
 
 export type TransactionToastComponentProps = ToastComponentProps & {
   message: string
-  txHash?: string
+  txHash?: Address
```

Comment:
I think probably just make this accept a string so you don't need to tell TS `txHash as Address` everywhere